scripts/bench.sh

```diff
@@ -5,9 +5,14 @@ MYDIR=$(realpath "$(dirname "$0")")
 cd "$MYDIR"
 cd ..
 
-NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly}
+NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly-2024-01-01}
 cargo +"$NIGHTLY_TOOLCHAIN" build --release --target wasm32-unknown-unknown -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
 
 export RPC_URL=http://localhost:8547
-cargo run --release -p benches
+
+# No need to run benchmark infrastructure with release.
+# Just compilation will take longer.
+# Contracts already built with release.
+cargo run -p benches
```

Comment:
If the contracts are already built with release, then this would skip compilation as well? I think the comment is a bit misleading, could we rephrase it?

If the reason is that `--release` has no impact here because only compiling the contracts with `--release` is what matters, then I think something like "No need to compile benchmarks with `--release` since gas measurement happens on the contract side".

Reply:
Yeah. You've got it correct. Will make it clear